### DISCUSSION TITLE
'linkquality' value for Aqara

### DISCUSSION THREAD

**Discussion by quiet2754** [Koenkk/zigbee2mqtt#6038] [Devices]
Hello

The `linkquality` value of my Aqara devices (Cube, door sensor, plug) are strange.
All values are the same and start to 0 and rise to 255 before restarting from 0, etc, etc, etc...

Examples:
`{"consumer_connected":false,"consumption":1.06,"current":0,"energy":1.06,"linkquality":18,"power":0,"state":"OFF","temperature":20,"voltage":226}`
`{"action":"wakeup","angle":10.31,"battery":100,"linkquality":18,"side":0,"voltage":3005}`
![image](https://user-images.githubusercontent.com/59255147/106514583-b952fc80-64d4-11eb-92c2-9f43a5daad29.png)

Does anybody has the same issue?

Best regards

**Comment by m0wlheld** [comment]
Do you have a Conbee /Raspee coordinator hardware? Then see my [issue](https://github.com/dresden-elektronik/deconz-rest-plugin/issues/4234) report. No solution so far.

**Reply by quiet2754** [reply]
Thank you for your reply.
I am using Conbee.

**Reply by m0wlheld** [reply]
Then you might consider adding a comment to the [issue](https://github.com/dresden-elektronik/deconz-rest-plugin/issues/4234) at deCONZ and close the issue here. It's an annoying bug that is not located in Zigbee2MQTT.
